src/Network/Mattermost/Types.hs

```diff
@@ -1222,6 +1222,24 @@ instance A.ToJSON Preference where
     , "value"    .= preferenceValue
     ]
 
+data FavoriteChannelPreference =
+    FavoriteChannelPreference { favoriteChannelId :: ChannelId
+                           , favoriteChannelShow :: Bool
```

Comment:
It looks the lines in this declaration need to be indented a bit more.

Reply:
fixed in the commit `f3fa8bb`